src/core/Table/TablePaginator.tsx

```diff
@@ -311,6 +331,11 @@ export const TablePaginator = (props: TablePaginatorProps) => {
         </IconButton>
       </div>
       <div className='iui-right'>
+        {localization.rowsPerPageLabel && currentWidth >= 1024 && (
```

Comment:
I am not sure if this should be 1024. Might be a bit too much. But 768 might be too little, idk